modules/microshift-nw-multus-add-pod.adoc

```diff
@@ -0,0 +1,175 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift_multiple_networks/microshift-cni-multus-using.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-nw-multus-add-pod_{context}"]
+= Adding a pod to an additional network
+
+You can add a pod to an additional network. At the time a pod is created, additional networks are attached to it. The pod continues to send normal cluster-related network traffic over the default network.
+
+If you want to attach additional networks to a pod that is already running, you must restart the pod.
+
+.Prerequisites
+
+* The OpenShift CLI (`oc`) is installed.
+* The cluster is running.
+* A network defined by a `NetworkAttachmentDefinition` object that you want to attach the pod to exists.
+
+.Procedure
+
+. Add an annotation to an existing `Pod` object. Only one of the following annotation formats can be used:
```

Comment:
Not sure about the "an existing" - annotation needs to be added to the .yaml file before creating the Pod. Existing Pod could be interpreted as Pod that's already running which we add annotations to (which I think might not be even possible, since only limited set of fields can be changed when Pod is already running)

Reply:
The OpenShift docs say "If you want to attach additional networks to a pod that is already running, you must restart the pod." So I assumed one can, but better to be conservative, agreed.